Note that type that type can be (#10674)
dotnet/docs: docs/framework/unmanaged-api/metadata/imetadatatables-getcolumninfo-method.md

  
 `ppName`  
 [out] A pointer to a pointer to the column name.  

>  

Reviewer: can you also change the following:
line 3 - update ms.date value
line 37 - remove 23 of the # from Parameters so it becomes an H2 instead of H4

## Remarks

> The returned column type falls within a range of values:

Reviewer 2: In order for any of the below constants or functions to resolve, the user appears required to #define _DEFINE_META_DATA_META_CONSTANTS so that the relevant code is included in the cor.h header. That seems important to include.

 Ideally the value returned by pType would have been defined as an enumeration, but instead it is defined as a set of constants:
const unsigned int iRidMax          = 63;
const unsigned int iCodedToken      = 64;   // base of coded tokens.
const unsigned int iCodedTokenMax   = 95;
const unsigned int iSHORT           = 96;   // fixed types.
const unsigned int iUSHORT          = 97;
const unsigned int iLONG            = 98;
const unsigned int iULONG           = 99;
const unsigned int iBYTE            = 100;
const unsigned int iSTRING          = 101;  // pool types.
const unsigned int iGUID            = 102;
const unsigned int iBLOB            = 103;

Do we have a place in the docs to document constants? Then this page could refer to a page that defined these constants, similar to what we would do for any enumeration. If not I think the inline table is fine, but I would separate out the helper function column and instead use some additional text to describe that the helper functions can assist in classifying the types into some useful groups. 

  - IsRidType() - returns TRUE for Rid types [0,63]
  - IsRidOrToken() - returns TRUE for Rid and Coded token types [0,95]
  - IsCodedTokenType() - returns TRUE for Coded token types [64,95]
  - IsFixedType() - returns TRUE for ISHORT, iUSHORT, iLONG, iULONG and iBYTE types [96, 100]
  - IsHeapType() - returns TRUE for iSTRING, iGUID, and iBLOB types [101,103] 

Maybe it was just me that got confused, but it took me a moment to figure out what the Helper function column was telling me. If everyone else was happy with the table as-is I can get on-board : )

| pType                    | Description   | Helper function                   |
|--------------------------|---------------|-----------------------------------|
| `0`..`iRidMax`<br>(0..63)   | Rid           | **IsRidType**<br>**IsRidOrToken** |
| `iCodedToken`..`iCodedTokenMax`<br>(64..95) | Coded token | **IsCodedTokenType** <br>**IsRidOrToken** |
| `iSHORT` (96)            | Int16         | **IsFixedType**                   |
| `iUSHORT` (97)           | UInt16        | **IsFixedType**                   |
| `iLONG` (98)             | Int32         | **IsFixedType**                   |
| `iULONG` (99)            | UInt32        | **IsFixedType**                   |
| `iBYTE` (100)            | Byte          | **IsFixedType**                   |
| `iSTRING` (101)          | String        | **IsHeapType**                    |
| `iGUID` (102)            | Guid          | **IsHeapType**                    |
| `iBLOB` (103)            | Blob          | **IsHeapType**                    |

> Values that are stored in the *heap* (*i.e.,* **IsHeapType** == true) can be read using:

Reviewer: Minor grammar suggestion

```suggestion
Values that are stored in the *heap* (that is, `IsHeapType == true`) can be read using:
```
